### PR TITLE
Classes encryption keys as 'small' items

### PR DESCRIPTION
**Pull request by SamHPurp**
Because why the hell are they bigger than the headsets they go into....

You couldn't put it in a box, but you could put it in a headset, and put that headset in a box....

:cl: Purpose2
tweak: Headset encryption keys are now 'small' items.
/ :cl:

### CODE REVIEW

**Comment by Xantholne** [comment]
This issue was pointed out in Msay even. They could storage implant their headset but now their key and it was very confusing as it makes literally zero sense. It uses a fucking clown car logic.

**Comment by SamHPurp** [comment]
@Xantholne psssh pass this infoz onto me to fix sooner.

File: code/game/objects/items/devices/radio/encryptionkey.dm [hunk 1]
```diff
@@ -5,6 +5,7 @@
 	icon = 'icons/obj/radio.dmi'
 	icon_state = "cypherkey"
 	item_state = ""
+	w_class = 1
```
Comment (Fox-McCloud) [inline code/game/objects/items/devices/radio/encryptionkey.dm]: Use defines.

Comment (SamHPurp) [inline code/game/objects/items/devices/radio/encryptionkey.dm]: @Fox-McCloud this is done

**Comment by Xantholne** [comment]
We only realized it some days before this

**Pull request merged**